modules/document_repository/php/editfile.class.inc

```diff
@@ -0,0 +1,171 @@
+<?php
+/**
+ * PHP Version 7
```

Comment:
missing a comment here.. I'm not sure what the difference between "edit" and "editfile" is supposed to be.